cmd/dagger/main.go

```diff
@@ -87,11 +89,20 @@ var rootCmd = &cobra.Command{
 		if err := os.Chdir(workdir); err != nil {
 			return err
 		}
+
+		t := analytics.New(analytics.DoNotTrack())
```

Comment:
@vito @sipsma Can you double check this here?

Reply:
We'll be tracking every `dagger` command invocation (e.g. even `dagger --help`). As long as that's the intention, LGTM